fragmentstest/src/main/java/org/odk/collect/fragmentstest/FragmentsTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.odk.collect.fragmentstest
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.matcher.RootMatchers.isDialog
+import org.hamcrest.Matcher
+
+/**
+ * Helpers for writing tests for Fragment that (hopefully) make interacting with [FragmentScenario]
+ * less troublesome.
+ */
+object FragmentsTest {
+
+    /**
+     * Creates a FragmentScenario using a test theme that supports AppCompat and MaterialComponents
+     * attributes but also won't explode (like MaterialComponents themes do).
+     *
+     * This uses [FragmentScenario.launch] here instead of [FragmentScenario.launchInContainer] as
+     * per docs at https://developer.android.com/guide/fragments/test#dialog.
```

Comment:
It's true that dialog fragments should be started using FragmentScenario.launch but when it comes to normal fragments using FragmentScenario.launchInContainer() is also possible. This class however is not just for dialogs right? but for all fragments, if so it should also have implementation for `launchInContainer()`.

Reply:
I was thinking for those we could just use `FragmentScenario` directly, but I could see that it might be nice to encapsulate everything into this object. What do you think?

Reply:
Yes it would be clearer if we had everything here and with comments.

Reply:
I gave it a try with `launch` and `launchInContainer` and it ended up feeling a bit unnecessary, as I don't think there is any reason not to use `FragmentScenario` directly for normal fragments. What I've done instead is renamed the helper object, so it's (hopefully) clear it's just for `DialogFragment`.